net-analyzer/suricata/suricata-7.0.6.ebuild

```diff
@@ -0,0 +1,223 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-1 luajit )
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit autotools flag-o-matic linux-info lua-single python-single-r1 systemd tmpfiles verify-sig
+
+DESCRIPTION="High performance Network IDS, IPS and Network Security Monitoring engine"
+HOMEPAGE="https://suricata.io/"
+SRC_URI="
+	https://www.openinfosecfoundation.org/download/${P}.tar.gz
+	verify-sig? ( https://www.openinfosecfoundation.org/download/${P}.tar.gz.sig )
+"
+
+LICENSE="GPL-2"
+SLOT="0/7"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="+af-packet af-xdp bpf control-socket cuda debug +detection geoip hardened hyperscan lua lz4 nflog +nfqueue redis systemd test"
+VERIFY_SIG_OPENPGP_KEY_PATH="/usr/share/openpgp-keys/openinfosecfoundation.org.asc"
+
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}
+	af-xdp? ( bpf )
+	bpf? ( af-packet )
+	lua? ( ${LUA_REQUIRED_USE} )
+"
+RDEPEND="
+	${PYTHON_DEPS}
+	acct-group/suricata
+	acct-user/suricata
+	dev-libs/jansson:=
+	dev-libs/libpcre2
+	dev-libs/libyaml
+	net-libs/libnet:*
+	net-libs/libnfnetlink
+	dev-libs/nspr
+	dev-libs/nss
+	$(python_gen_cond_dep 'dev-python/pyyaml[${PYTHON_USEDEP}]')
+	>=net-libs/libhtp-0.5.48
+	net-libs/libpcap
+	sys-apps/file
+	sys-libs/libcap-ng
+	af-xdp?	( net-libs/xdp-tools )
+	bpf?       ( dev-libs/libbpf )
+	cuda?      ( dev-util/nvidia-cuda-toolkit )
+	geoip?     ( dev-libs/libmaxminddb:= )
+	hyperscan? ( dev-libs/vectorscan:= )
+	lua?       ( ${LUA_DEPS} )
+	lz4?       ( app-arch/lz4 )
+	nflog?     ( net-libs/libnetfilter_log )
+	nfqueue?   ( net-libs/libnetfilter_queue )
+	redis?     ( dev-libs/hiredis:= )
```

Comment:
Let's not keep this unconventional aligning style, and replace them with one single space please.